packages/client-search/src/types/Log.ts

```diff
@@ -68,4 +68,14 @@ export type Log = {
    * Index name of the log
    */
   readonly index: string;
+  
+  /**
+   * internal queries performed for this query.
+   */
+  readonly inner_queries: ReadonlyArray<{
+    indexName: string;
```

Comment:
Can you add a small description for each field on `inner_queries`? It's really about keeping  beautiful documentation along side all fields.

Reply:
They don't have a description in the api ref, so I don't know what to put

Reply:
What I wrote in the doc: 

Contains an object for each performed query with the indexName, queryID, offset, and userToken.